api/README.md

```diff
@@ -718,3 +722,31 @@ empty string, which will yield all test results for the selected runs.
 ```
 
 </details>
+
+## Badges _(Beta)_
+
+### /api/badge
+
+Generates and serves an `image/svg+xml` response of an [shields.io](https://shields.io) badge, for the overall status of a path (folder), for a given product.
+
+__An important note on gamification:__
+
+As noted on the website,
+
+>_wpt.fyi is a work in progress. The reported results do not necessarily reflect the true capabilities of each web browser, so they should not be used evaluate or compare feature support._
+
+As such, we discourage contributors from gamification of overall pass rates, as this can inhibit contribution of (new) failing tests, which can be important in revealing interoperability problems.
+
+__Parameters__
+
+__`product`__ : The product to compute the summary badge for, e.g. `chrome`, or `firefox-60`, or `edge[experimental]`.
+
+__`path`__ : The test path to filter by, e.g. `/webdriver`.
+
+<details><summary><b>Example badge</b></summary>
+
+Latest chrome experimental run:
+
+![webdriver badge](https://staging.wpt.fyi/api/badge?product=chrome&path=/webdriver)
```

Comment:
As a preview for what this will render like, I added a commit that uses this PR's staging environment here:
https://github.com/web-platform-tests/wpt.fyi/pull/674/commits/1ed6008189f34fb79f90d0df172e18d54d0c473e?short_path=d4b47e8#diff-d4b47e85ce206fdbe458cb599edf2a90

(View it in rich mode)